### ISSUE TITLE
[Accessibility]A11y_PowershellEditorServices_Installation_Screenreader:Narrator does not read the content written under the Tab item (Details,Contributions,Changelog,Dependencies).

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61717]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that what narrator reads for the content written under the Tab items("Details,Contributions,Changelog,Dependencies")

**Actual:**
Narrator does not read the content written under the Tab item (Details,Contributions,Changelog,Dependencies).
 
**Expected:**
Narrator should read the content written under the Tab items.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User unable know what contents written under the tab items.

**MAS Reference:**
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference**
[2731288_A11y_PowershellEditorServices_Installation_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2510496/2731288_A11y_PowershellEditorServices_Installation_Screenreader.pptx)

**Comment by isidorn** [comment]
Narrator is not supported
